src/exprs.c

```diff
@@ -919,9 +919,19 @@ Obj             EvalPermExpr (
 **
 **  'EvalListExpr'  just  calls 'ListExpr1'  and  'ListExpr2' to evaluate the
 **  list expression.
+**
+**  The 'safe' argument to ListExpr1 and ListExpr2 should be set to 1 when
+**  the list expression may be modified when its elements are being
+**  constructed. This can occur when an element of the list refers to ~.
+**
+**  When 'safe' is 1, we use a slower code path which ensures the list is
```

Comment:
Hmm, when I was reading the code below, I constantly had to refer back to this comment, because I could not remember whether `safe` meant: either "we are safe; it is safe to assume that the list won't be messed with", or perhaps "play it safe; use safe code which works even if the list gets messed with".

Since these are opposite, that's a bit of a problem, IMHO. Is it just me? Perhaps we can find a better name. Say `protectAgainstListModification` or `safelyGrowList`, `safelyExtendList`; or perhaps reverse the value, and say `assumeListIsPlist` or ...? Sorry, no great idea.

Reply:
I could just call it `tildeInUse`, seeing as that's all we use it for :)

Reply:
Yes, that actually sounds very good to me!